package.json

```diff
@@ -83,6 +83,7 @@
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.1",
     "ember-load-initializers": "^2.1.2",
+    "ember-named-blocks-polyfill": "^0.2.5",
```

Comment:
I temporarily installed a polyfill so that I could use named blocks (landed in Ember v3.25) to simplify code, while satisfying the `ember-lts-3.24` scenario.

We can remove the polyfill when we drop support for Ember 3.24 LTS.